src/extension-runners/firefox-android.js

```diff
@@ -400,7 +400,13 @@ export class FirefoxAndroidExtensionRunner {
       selectedAdbDevice
     );
 
-    if (androidVersion < 21) {
+    if (typeof androidVersion !== 'number' || isNaN(androidVersion)) {
+      throw new WebExtError(`Invalid Android version: ${androidVersion}`);
+    }
+
+    log.debug(`Detected Android version ${androidVersion}`);
+
+    if (androidVersion < 23) {
```

Comment:
Could you adjust [this comment](https://github.com/rpl/web-ext/blob/a68cb531f07eed5bd2dcb223a16ad6a8f061f2f8/src/extension-runners/firefox-android.js#L416)? It still says android-21

Reply:
yeah, definitely, I should have run a grep :-)